src/parser/ast.rs

```diff
@@ -304,6 +307,12 @@ pub enum TraitAdaptation {
     },
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub struct Attribute {
+    pub span: Span,
```

Comment:
we don't do this for other nodes, but we should start doing it.

Reply:
Yeah, having some sort of `meta` property that contains the span, and any other info would be useful. Comments perhaps